cfn-resources/encryption-at-rest/README.md

```diff
@@ -2,17 +2,65 @@
 
 ## Description
 This resource allows administrators to enable, disable, configure, and retrieve the configuration for Encryption at Rest.
+# Attributes & Parameters
 
-## Parameters
-`ProjectId` *(required)* : The unique identifier of the project.<br>
-`AwsKms.AccessKeyID` *(required)* : The IAM access key ID with permissions to access the customer master key specified by customerMasterKeyID.<br>
-`AwsKms.CustomerMasterKeyID` *(required)* : The AWS customer master key used to encrypt and decrypt the MongoDB master keys.<br>
-`AwsKms.Enabled` *(required)* : The IAM secret access key with permissions to access the customer master key specified by customerMasterKeyID.<br>
-`AwsKms.SecretAccessKey` *(required)* : Specifies whether Encryption at Rest is enabled for an Atlas project. To disable Encryption at Rest, pass only this parameter with a value of false. When you disable Encryption at Rest, Atlas also removes the configuration details.<br>
-`AwsKms.Region` *(required)* : The AWS region in which the AWS customer master key exists.<br>
-`ApiKeys` *(required)* : The private and public keys of the MongoDB Atlas organization or project.<br>
+Please consult the [Resource Docs](docs/README.md)
+
+## Unit Testing Locally
+
+The local tests are integrated with the AWS `sam local` and `cfn invoke` tooling features:
+
+```
+sam local start-lambda --skip-pull-image
+```
+then in another shell:
+```bash
+repo_root=$(git rev-parse --show-toplevel)
+source <(${repo_root}/quickstart-mongodb-atlas/scripts/export-mongocli-config.py)
+set or export below environment variables 
+export KMS_KEY=<<CustomerMasterKeyID>>
+export KMS_ROLE=<<RoleID>>
+export KMS_REGION=<<key region>>
+cd ${repo_root}/cfn-resources/encryption-at-rest
+./test/encryptionatrest.create-sample-cfn-request.sh YourProjectID > test.request.json 
+echo "Sample request:"
+cat test.request.json
+cfn invoke CREATE test.request.json 
+cfn invoke DELETE test.request.json 
+```
+
+Both CREATE & DELETE tests must pass.
 
 ## Installation
-    $ make
-    $ cfn submit
-    ...
\ No newline at end of file
+TAGS=logging make
+cfn submit --verbose --set-default
+
+## Cloudformation Examples
+
+Please use the [CFN Template](test/encryptionatrest.sample-template.yaml)
+
+## Integration Testing w/ AWS
+
+The [/quickstart-mongodb-atlas/scripts/launch-x-quickstart.sh](launch-x-quickstart.sh) script
```

Comment:
This link doesn't work, can you double check to bring user to correct GitHub page

Reply:
This issue is fixed